packages/xo-server/src/collection/redis.mjs

```diff
@@ -172,16 +175,7 @@ export default class Redis extends Collection {
         }
 
         const key = `${prefix}:${id}`
-        const props = {}
-        for (const name of Object.keys(model)) {
-          if (name !== 'id') {
-            const value = model[name]
-            if (value !== undefined) {
-              props[name] = String(value)
-            }
-          }
-        }
-        const promises = [redis.del(key), redis.hSet(key, props)]
+        const promises = [redis.del(key), redis.set(key, JSON.stringify(model))]
```

Comment:
I think `JSON.stringify` do not return a promise

Reply:
But `redis.set` does :slightly_smiling_face: